job.py

```diff
@@ -174,16 +174,16 @@ def runReform(target_dir, ref_fasta, ref_gff, timestamp, position, chrom, in_fas
 
 
 def send_email(email, timestamp):
-    # calculate 72h DDL
-    deadline = datetime.now() + timedelta(hours=72)
+    # calculate 168h DDL
+    deadline = datetime.now() + timedelta(hours=168)
     deadline_str = deadline.strftime('%Y-%m-%d %H:%M:%S')
 
     with j.app_context():
         subject = f"Reform Results - Download Deadline: {deadline_str}"
         msg = Message(subject, sender='reform@nyu.edu', recipients=[email])
         msg.html = f"""Reform job complete. 
                        <a href='https://reform.bio.nyu.edu/download/{timestamp}'>Click here to download results</a>. 
-                       The file will be available for the next 72 hours. 
+                       The file will be available for the next 1 week (168 hours). 
                        The deadline to download the file is {deadline_str}. 
```

Comment:
Change to "The deadline to download the file is {deadline_str}." which should look like
> The deadline to download the file is June 12, 2024.
We will delete the file June 13th at 1am.